tests/test_logistic.py

```diff
@@ -159,27 +156,19 @@ def test_pdf(xscale: Scale):
 
     ## Make sure it integrates to 1
     _xs = xscale.denormalize_points(np.linspace(0, 1, 100))
-    densities_logistic = np.array([float(ergoLogistic.pdf(x)) for x in _xs])
-    densities_mixture = np.array([float(ergoLogisticMixture.pdf(x)) for x in _xs])
-    auc_logistic = float(trapz(densities_logistic, x=_xs))
-    auc_mixture = float(trapz(densities_mixture, x=_xs))
-    assert (
-        1
-        == pytest.approx(auc_logistic, abs=0.03)
-        == pytest.approx(auc_mixture, abs=0.03)
-    )
+    auc_logistic = float(trapz(ergoLogistic.pdf(_xs), x=_xs))
+    auc_mixture = float(trapz(ergoLogisticMixture.pdf(_xs), x=_xs))
+    assert 1 == pytest.approx(auc_logistic, abs=0.03)
```

Comment:
Pytest doesn't like comparing two approximate arrays, the following throws an error:
`pytest.approx([]) == pytest.approx([])`

My vague sense is that directly comparing approximate objects is probably not ideal since it's not clear to me how the tolerances would be combined.

Reply:
(note the array comparison isn't actually a problem in these lines but is elsewhere).